dataactcore/utils/fileBOC.py

```diff
@@ -0,0 +1,293 @@
+from collections import OrderedDict
+from sqlalchemy import or_, and_, func, null, values, column
+from sqlalchemy.sql.expression import case, literal
+
+from dataactcore.models.domainModels import GTASBOC
+from dataactcore.models.lookups import PUBLISH_STATUS_DICT
+from dataactcore.models.stagingModels import PublishedObjectClassProgramActivity
+from dataactcore.models.jobModels import Submission
+
+fileb_model = PublishedObjectClassProgramActivity
+boc_model = GTASBOC
+
+mapping = OrderedDict([
+    ('display_tas', ['TAS']),
+    ('allocation_transfer_agency', ['AllocationTransferAgencyIdentifier']),
+    ('agency_identifier', ['AgencyIdentifier']),
+    ('beginning_period_of_availa', ['BeginningPeriodOfAvailability']),
+    ('ending_period_of_availabil', ['EndingPeriodOfAvailability']),
+    ('availability_type_code', ['AvailabilityTypeCode']),
+    ('main_account_code', ['MainAccountCode']),
+    ('sub_account_code', ['SubAccountCode']),
+    ('object_class', ['ObjectClass']),
+    ('by_direct_reimbursable_fun', ['ByDirectReimbursableFundingSource']),
+    ('disaster_emergency_fund_code', ['DisasterEmergencyFundCode']),
+    ('prior_year_adjustment', ['PriorYearAdjustment']),
+    ('begin_end', ['BeginningEndingIndicator']),
+    ('reporting_fiscal_year', ['ReportingFiscalYear']),
+    ('reporting_fiscal_period', ['ReportingFiscalPeriod']),
+    ('ussgl_account_num', ['USSGLAccountNumber']),
+    ('dollar_amount_gtas', ['DollarAmountGTAS']),
+    ('dollar_amount_broker', ['DollarAmountDataBroker']),
+    ('dollar_amount_diff', ['DollarAmountGTASSubDataBroker']),
+    ('file_b_rows', ['FileBRows'])
+])
+db_columns = [key for key in mapping]
```

Comment:
I think since Python3.7, dicts have preserved their order and I confirmed locally you can just use a dict just the same. I was gonna suggest updating it but realized it's just following the convention of file generations. 👍 Just another tech debt consideration down the road.